src/views/notification/Edit.vue

```diff
@@ -0,0 +1,286 @@
+                              <div v-if="this.validation_error?.type != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.type[0] }}
+                              </div>
+                              {{ this.notification?.type }}
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12">
+                            <div class="flex items-center">
+                              <div class="border-l-2 border-theme-1 pl-4">
+                                <a href="" class="font-medium">
+                                  Has the notification already been seen?
+                                </a>
+                                <div class="text-gray-600">
+                                  Select this option to mark the notification as read.
+                                </div>
+                              </div>
+                              <input class="form-check-switch ml-auto" type="checkbox" v-model="this.notification.seen">
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12">
+                            <hr class="my-5">
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label for="update-icon-notification" class="form-label">
+                                Icon
+                              </label>
+                              <input
+                                id="update-icon-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.icon != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Icon"
+                                v-model="this.notification.icon"
+                                :disabled="parseInt(this.notification?.type) !== 1"
+                              />
+                              <div v-if="parseInt(this.notification?.type) !== 1" class="text-theme-6 mt-2 mb-4">
+                                User Type is selected!
+                              </div>
+                              <div v-if="this.validation_error?.icon != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.type[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label class="form-label">
+                                User
+                              </label>
+                              <TailSelect
+                                :class="'form-control' + (this.validation_error?.target_id != null ? ' border-theme-6' : '')"
```

Comment:
`!= null` is unnecessary, questionmark-syntax is taking a valid value as true by default

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/d168bab06f57a70137f9e8dab39812eb5b6ac6a9